### PR TITLE
Finished # Tips

### PR DESCRIPTION
**Pull request by S1ngS1ng**
**DO NOT MERGE**

- Needs rebase after #13

### CODE REVIEW

**Comment by wsdjeg** [comment]
@S1ngS1ng #13 is merged, and the links have beed fixed.

File: README.md [hunk 1]
```diff
@@ -60,8 +60,8 @@
 
 ## [技巧](#技巧-1)
 
-- [聪明的使用 n 和 N](#聪明的使用-n-和-n)
-- [聪明的使用命令行历史](#聪明的使用命令行历史)
+- [聪明地使用 n 和 N](#聪明地使用-n-和-n)
+- [聪明地使用命令行历史](#聪明地使用命令行历史)
```
Comment (S1ngS1ng) [inline README.md]: Some minor updates here. This is more accurate IMHO.

File: README.md [hunk 2]
```diff
@@ -72,8 +72,9 @@
 - [根据模式改变光标类型](#根据模式改变光标类型)
 - [防止水平滑动的时候失去选择](#防止水平滑动的时候失去选择)
 - [重新载入保存文件](#重新载入保存文件)
-- [智能当前行](#智能当前行)
+- [更加智能的当前行高亮](#更加智能的当前行高亮)
 - [更快的关键字补全](#更快的关键字补全)
+- [改变颜色主题的默认外观](#改变颜色主题的默认外观)
```
Comment (S1ngS1ng) [inline README.md]: This one was missing.

File: README.md (hunk 1 above)
Comment (wsdjeg) [inline README.md]: yeah, It is has been done in your PR?

Comment (S1ngS1ng) [inline README.md]: @wsdjeg Yep. I think so.

[Line 63](https://github.com/wsdjeg/vim-galore-zh_cn/pull/15/files#diff-04c6e90faac2675aa89e2176d2eec7d8R1413)

[Line 64](https://github.com/wsdjeg/vim-galore-zh_cn/pull/15/files#diff-04c6e90faac2675aa89e2176d2eec7d8R1424)

**Pull request merged**